src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingFatalErrorHelpers.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.ErrorReporting;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal static class UnitTestingFatalErrorHelpers
```

Comment:
💭 It seems like this is an accessor type

```suggestion
    internal static class UnitTestingFatalErrorAccessor
```